build/windows/demo/local.yaml

```diff
@@ -12,6 +12,6 @@ logging:
   level: Info
   directory: log
   filename: remote_server.log
-  max_file_count: 5
-  max_file_size: 10
+  max_file_count: 10
```

Comment:
have changed the defaults, as I think that 10MB limit for the log files is excessive!